### PR TITLE
Merge upstream

### PR DESCRIPTION
**Pull request by honza**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+on:
```
Comment (elfosardo) [inline .github/workflows/release.yaml:1]: we probably don't need this

File: Makefile [hunk 2]
```diff
@@ -271,3 +271,17 @@ vendor:
 	cd pkg/hardwareutils; go mod vendor
 	cd hack/tools; go mod vendor
 	go mod vendor
+
```
Comment (elfosardo) [inline Makefile:274]: this change also does not apply to us

File: docs/releasing.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# Releasing
```
Comment (elfosardo) [inline docs/releasing.md:1]: same we can remove this

File: hack/tools/release_notes.go [hunk 4]
```diff
@@ -0,0 +1,204 @@
+//go:build tools
```
Comment (elfosardo) [inline hack/tools/release_notes.go:1]: this can also be skipped

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (dtantsur) [reply Makefile:274]: I'm not sure we should diverge from the upstream Makefile. But we should make sure the github workflow will not run.

**Pull request merged**